### DISCUSSION TITLE
any examples communicating with AWS App Config extension?

### DISCUSSION THREAD

**Discussion by evbo** [awslabs/aws-sdk-rust#856] [Q&A] [answered]
I found a github from 2 years ago and am wondering if there's any new recommendations for integrating with AppConfig via lambda extensions?

https://github.com/ymwjbxxq/rust_aws_app_config/tree/main

**Comment by Velfi** [comment, marked as answer]
We don't appear to have an example of this in the [examples repo](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/rust_dev_preview) but I'm sure if you submit an issue to them, they'll get to it eventually. You might also try asking the folks in the [aws-lambda-rust-runtime](https://github.com/awslabs/aws-lambda-rust-runtime) repo.

The example you linked looks reasonable at a glance, but I don't have familiarity with AppConfig myself, so take that with a grain of salt.

**Comment by ymwjbxxq** [comment]
Maybe this can help you
https://serverlessland.com/patterns?services=appconfig&framework=SAM&language=Rust
